.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,53 @@
+---
+name: 🐛Bug Report🐛
+about: Report an Issue with the JDBC Driver
+title: "[BUG]"
+labels: 'bug, untriaged, Beta'
+assignees: ''
+---
+
+## Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+- [ ] **Is this bug/limitation described in the current 
```

Comment:
I think the wording of this could be improved since right now it feels like you need to check off that it **is** in the list of limitations rather than not.